src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationHandler.cs

```diff
@@ -24,7 +24,12 @@ public LinkedInAuthenticationHandler([NotNull] HttpClient client)
 
         protected override async Task<AuthenticationTicket> CreateTicketAsync([NotNull] ClaimsIdentity identity,
             [NotNull] AuthenticationProperties properties, [NotNull] OAuthTokenResponse tokens) {
-            var request = new HttpRequestMessage(HttpMethod.Get, Options.UserInformationEndpoint);
+            var endpoint = string.Empty;
```

Comment:
Good point, the user could clear the pre-set fields out. Changing this to throw an exception if there are no fields present.